### ISSUE TITLE
Disonnecting during transcription

### ISSUE THREAD

**Issue by Ghostkwebb** [abdeladim-s/subsai#46]
So I was trying to do [this](https://github.com/abdeladim-s/subsai/discussions/29#discussioncomment-5797601) and this problem kept on coming and I don't know how to fix it or if there is a workaround to this.

https://github.com/abdeladim-s/subsai/assets/61558762/d9164cdf-7f9d-4994-a1ff-8a5b3bdbd0f0

**Comment by abdeladim-s** [comment]
Could you please create a new virtual environment and give a try a now? I think it will be solved now!

**Comment by Ghostkwebb** [comment]
I tried again and reinstalled it because the same thing happened before reinstalling.

https://github.com/abdeladim-s/subsai/assets/61558762/c63cbd8e-9b8c-4819-90c9-42afa158ed5e

**Comment by abdeladim-s** [comment]
I think I found the cause of the problem. 
The download url of `whisper.cpp` ggml models has been changed, and that makes a 404 page to be downloaded instead of the models.  

* So first, make sure to delete the old downloaded "models" from your user directory (On Linux, it should be `/home/user/.local/share/pywhispercpp/models/` - On Mac, I am not sure, but maybe something like `/Users/user/Library/Application Support/pywhispercpp/models/`) 
* After that reinstall `subsai` and make sure you have `pywhsipercpp==1.1.1`. 
* When you run it again, it will download the correct models, that's why it is necessary to delete the old models folder first. 

Let me know if you find any other issue ?
